### PR TITLE
Rj starting on UI

### PR DESCRIPTION
**Pull request by jrzricardo**
- Added navbar

- Updated dropdown menu links

- Added notice messages

- Created user authentication conditions.

### CODE REVIEW

File: app/views/layouts/application.html.erb
Comment (APatel-AI) [inline app/views/layouts/application.html.erb:1]: Looks good

File: lib/tasks/dev.rake
Comment (APatel-AI) [inline lib/tasks/dev.rake:1]: Great job on generating the sample_data using Faker.

**Review by APatel-AI — APPROVED** [review]

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:2]: Nice!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,13 +6,27 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:9]: Great job adding your CDN assets to a partial! This helps keep your layout file clean :)

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <a href="/photos" class="dropdown_item">Photos</a>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:14]: Remember we can start using helper methods now like link_to :)

**Review by JoseMDO — APPROVED** [review]
Looks great Ricardo!

**Pull request merged**